validator/client/propose_test.go

```diff
@@ -194,7 +194,7 @@ func TestProposeBlock_BlocksDoubleProposal(t *testing.T) {
 	m.validatorClient.EXPECT().DomainData(
 		gomock.Any(), // ctx
 		gomock.Any(), // epoch
-	).Times(2).Return(&ethpb.DomainResponse{SignatureDomain: make([]byte, 32)}, nil /*err*/)
+	).AnyTimes().Return(&ethpb.DomainResponse{SignatureDomain: make([]byte, 32)}, nil /*err*/)
```

Comment:
Any reason this was changed? A precise number of times is a more rigid test. 

> AnyTimes allows the expectation to be called 0 or more times.

Not sure this is desired.